src/main/java/org/czekalski/petapiintegration/apiClient/v1/dto/AddressDto.java

```diff
@@ -0,0 +1,77 @@
+
+package org.czekalski.petapiintegration.apiClient.v1.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.*;
+
+
+public class AddressDto {
+
+
+
+    private String address1;
```

Comment:
Tak jak pozostałe pola to też może być final
```suggestion
    private final String address1;
```